### DISCUSSION TITLE
How wide does web browser support range of Swiper go?

### DISCUSSION THREAD

**Discussion by ngdangtu-vn** [nolimits4web/swiper#4697] [Q&A]
As the very first line goes like this:

> Swiper is not compatible with all platforms, it is a modern touch slider which is focused only on modern apps/platforms to bring the best experience and simplicity.

This makes me wonder what is the detail limitation for this library? My website aims to 95 → 96% browser support range.

**Comment by Andrew-Marks-Trisept** [comment]
I assume it basically means no IE, but I can't find this answer anywhere.
